buildscripts/verify-build.sh

```diff
@@ -68,6 +68,36 @@ function start_minio_erasure_sets()
     echo "$minio_pid"
 }
 
+function start_minio_dist_erasure_sets_ipv6()
```

Comment:
make: *** [verify] Error 1
The command "make verify" exited with 2.

Reply:
@balamurugana looks like your gateway setup failed

```
Testing in Distributed Erasure setup as sets with ipv6
{"name": "mc", "duration": "1178", "function": "test_make_bucket", "status": "PASS"}
{"name": "mc", "duration": "47", "function": "test_make_bucket_error", "status": "PASS"}
{"name": "mc", "duration": "697", "function": "test_put_object", "status": "PASS"}
{"name": "mc", "duration": "18", "function": "test_put_object_error", "status": "PASS"}
{"name": "mc", "duration": "405", "function": "test_put_object_0byte", "status": "PASS"}
{"name": "mc", "duration": "280", "function": "test_put_object_with_storage_class", "status": "PASS"}
{"name": "mc", "duration": "116", "function": "test_put_object_with_storage_class_error", "status": "PASS"}
{"name": "mc", "duration": "1152", "function": "test_put_object_multipart", "status": "PASS"}
{"name": "mc", "duration": "475", "function": "test_get_object", "status": "PASS"}
{"name": "mc", "duration": "2537", "function": "test_get_object_multipart", "status": "PASS"}
{"name": "mc", "duration": "1063", "function": "test_presigned_post_policy_error", "status": "PASS"}
{"name": "mc", "duration": "1345", "function": "test_presigned_put_object", "status": "PASS"}
{"name": "mc", "duration": "1134", "function": "test_presigned_get_object", "status": "PASS"}
{"name": "mc", "duration": "476", "function": "test_cat_object", "status": "PASS"}
{"name": "mc", "duration": "2013", "function": "test_mirror_list_objects", "status": "PASS"}
{"name": "mc", "duration": "2257", "function": "test_mirror_list_objects_storage_class", "status": "PASS"}
{"name": "mc", "duration": "3455", "function": "test_find", "status": "PASS"}
{"name": "mc", "duration": "2079", "function": "test_find_empty", "status": "PASS"}
{"name": "mc", "duration": "125", "function": "test_config_host_add", "status": "PASS"}
{"name": "mc", "duration": "71", "function": "test_config_host_add_error", "status": "PASS"}
Testing in Gateway S3 setup
FAILED
Makefile:66: recipe for target 'verify' failed
make: *** [verify] Error 1
```